### PR TITLE
docs: create sidebar entries for Mend-hosted app secrets pages

### LINKED ISSUE #30711: Move Mend-app only pages into "Mend-hosted apps" section in the sidebar
### Describe the proposed change(s).

Move these pages into a top-level "Mend-hosted apps" section of the sidebar:

- [x] App secrets
- [x] Migrating secrets

The pages should go in this part of the sidebar:

Renovate docs sidebar, red box around text Mend-hosted apps, top-level item, between Configuration and Key Concepts

Relevant discussion:

- #30696

### PR DESCRIPTION
**Pull request by justo-mend**
## Changes

Closes #30711

## Context

Provides an easy way to find pages related to the Mend-hosted apps.
Includes links to the App Secrets and Secrets Migration pages.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — APPROVED** [review]

**Comment by justo-mend** [comment]
All checks have passed. Can somebody please merge this PR. Thanks.

File: docs/usage/.pages [hunk 1]
```diff
@@ -9,6 +9,7 @@ nav:
       - 'Self-hosted': 'self-hosted-configuration.md'
       - 'Presets': 'config-presets.md'
       - 'Validation': 'config-validation.md'
+  - ... | mend-hosted
```
Comment (HonkingGoose) [inline docs/usage/.pages:12]: Please use the `Closes` keyword [^1] in your top post. That way, GitHub automatically closes the issue when the PR is merged.

Here's the old text, and the new text that you should use:

```diff
- Addresses #30711
+ Closes #30711
```

[^1]: [GitHub Docs, Using keywords in issues and pull requests](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests)

**Review by HonkingGoose — COMMENTED** [review]
Please update your top post, so it uses the `Closes` keyword. :wink:

File: docs/usage/.pages (hunk 1 above)
Comment (justo-mend) [reply docs/usage/.pages:12]: Ah! Thanks for the info. Will do in future.

Comment (justo-mend) [reply docs/usage/.pages:12]: Updated as suggested. Thanks

**Comment by justo-mend** [comment]
Can someone please merge this PR?

**Review by viceice — APPROVED** [review]

**Comment by justo-mend** [comment]
Please please - can someone merge this PR?

**Pull request merged**